src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1439,6 +1634,58 @@ private static unsafe (string, BlittableJsonReaderObject) GetCurrentItem(Transac
             return (key, doc);
         }
 
+        public BlittableJsonReaderObject GetCertificateByThumbprint(TransactionOperationContext context, string thumbprint)
+        {
+            var certs = context.Transaction.InnerTransaction.OpenTable(CertificatesSchema, CertificatesSlice);
+
+            using (Slice.From(context.Allocator, thumbprint, out var thumbprintSlice))
```

Comment:
I think you need to call `ToLower` here, no?

Reply:
yes, I went over all the occurrences of thumbprint as a key and changed them to lower